test/drenv/tools.py

```
@@ -0,0 +1,116 @@
+    """Install kustomize wrapper script in install_dir."""
+    os.makedirs(install_dir, exist_ok=True)
+    path = os.path.join(install_dir, "kustomize")
+    root = repo_root()
+    with open(path, "w", encoding="utf-8") as f:
+        f.write(
+            f"""#!/bin/sh
+cd {root!r} || exit 1
+exec go tool kustomize "$@"
+"""
+        )
+    os.chmod(path, 0o755)
```

Ruff (0.15.17)

[error] `os.chmod` setting a permissive mask `0o755` on file or directory

(S103)